### PR TITLE
Change logger name

### PR DESCRIPTION
**Pull request by shreyash2106**

### CODE REVIEW

File: src/agrag/args.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 CURRENT_DIR = os.path.dirname(__file__)
 
+from agrag.constants import LOGGER_NAME
+
 logger = logging.getLogger("AutoGluon-RAG-logger")
```
Comment (cheungdaven) [inline src/agrag/args.py:11]: should not it be replaced?

Comment (shreyash2106) [reply src/agrag/args.py]: Oops, fixed!

**Review by cheungdaven — APPROVED** [review]
LGTM

**Comment by shreyash2106** [comment]
> LGTM

@cheungdaven you will have to merge the PR since I don't have permissions anymore.

**Pull request merged**